examples/tests/valid/structs.w

```diff
@@ -21,6 +21,9 @@ let y = B {
   }
 };
 
+// successful struct access
+assert(x.field0 == "Sup");
+
```

Comment:
There are 2 commented out tests in this file with TODO: based on this card. Can we uncomment those?

Reply:
Oops, totally missed those. Looking now

Reply:
Fixed